ablog/templates/catalog.html

```diff
@@ -1,5 +1,12 @@
 {%- extends "page.html" %} {% block body %} {% for collection in catalog %} {%
 if collection %}
+{% macro postlink(post) %}

```

Comment:
I could not for the life of me figure out how to import this macro from one file to another, apologies for the duplication :-/

Reply:
I don't actually know how/if it is possible to import another macro, so I have no issue with the duplication.